testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -243,21 +244,20 @@ export default PageView.extend({
 
   renderUninstallSurvey(evt) {
     evt.preventDefault();
+
     app.sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
       eventLabel: 'Disable Experiment'
     });
+
+    const queryParams = querystring.stringify({ref: 'disable', experiment: this.model.title});
+
     this.renderSubview(new FeedbackView({
       id: 'disabled-feedback',
       experiment: this.model.url,
       title: 'feedbackUninstallTitle',
-      questions: [
-        { value: 'broken', title: 'feedbackUninstallAnswerBroken' },
-        { value: 'dislike', title: 'feedbackUninstallAnswerDislike' },
-        { value: 'notuseful', title: 'feedbackUninstallAnswerNotUseful' },
-        { value: 'other', title: 'feedbackUninstallAnswerOther' }
-      ],
+      surveyUrl: `http://meandave.com/faux_survey/?${queryParams}`,
```

Comment:
I could be wrong, but my impression is that the SurveyGizmo URLs are going to be altogether different for each experiment. Assuming that's the case, they'll need to be supplied in new fields on the Experiment model, served up in the API results, and pulled in dynamically. The query parameters are needed to grab additional user feedback like the star ratings, but beyond that I think each experiment gets its own survey.